certs/ed25519/ca-ed25519-key.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I don't see the new files in this PR being added to include.am(s).  Did I miss those?

Why is the PEM header "RSA KEY" while the file name is ed25519?